releasenotes/notes/update-figure-name-2db258c30ffe9912.yaml

```diff
@@ -0,0 +1,12 @@
+---
+other:
+  - |
+    Figure names have been updated to include qubit indices up to the first five physical qubits in
+    the experiment, with format ``StandardRB_Q0_Q1_Q2_Q3_Q5_b4f1d8ad.svg``. For composite
```

Comment:
Device components can be resonators instead of qubits

Reply:
Good point, fixed.